runtime/gc_modron_startup/mmhelpers.cpp

```diff
@@ -322,7 +327,39 @@ j9gc_modron_getConfigurationValueForKey(J9JavaVM *javaVM, UDATA key, void *value
 		*((UDATA *)value) = extensions->compressObjectReferences();
 		keyFound = TRUE;
 		break;
-		
+	case j9gc_modron_configuration_heapBase:
+		if (extensions->isVLHGC()) {
+			*((UDATA *)value) = (UDATA) extensions->heapRegionManager->getFirstTableRegion()->getLowAddress();
+			keyFound = TRUE;
+		} else {
+			*((UDATA *)value) = 0;
+			keyFound = false;
+		}
+		break;
+	case j9gc_modron_configuration_heapRegionShift:
+		if (extensions->isVLHGC()) {
+			*((UDATA *)value) = extensions->heapRegionManager->getRegionShift();
+			keyFound = TRUE;
+		} else {
+			*((UDATA *)value) = 0;
+			keyFound = false;
+		}
+		break;
+	case j9gc_modron_configuration_heapRegionStateTable:
+#if defined(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD)
+		if (extensions->isVLHGC() && (NULL != extensions->heapRegionStateTable)) {
```

Comment:
This could be a check for `extensions->isConcurrentCopyForwardEnabled()`. Would that be better?

Reply:
Fixed this.

Reply:
I would prefer the check you suggested.